packages/luna/src/components/MultiSelect/MultiSelect.tsx

```diff
@@ -110,7 +110,7 @@ export const MultiSelect = <T,>({
                       >
                         <Typography
                           element="body"
-                          className="tw-text-natural-50"
+                          className="tw-text-natural-50 tw-whitespace-nowrap"
```

Comment:
I am curious, why do we need this classname?

Reply:
if there wasn't enough space the text will be in 2 lines